src/components/10-atoms/input-text/index.js

```diff
@@ -248,6 +252,31 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
     this.nativeInput = this.querySelector('input');
   }
 
+  _formatCurrency(value) {
+    const { currency, type } = this;
+
+    if (currency && type === 'text' && !this.currencyFormatter) {
+      // just create a new Intl if it does not exist
+      this.currencyFormatter = new Intl.NumberFormat('de-CH', {
+        style: 'currency',
+        currency,
+      });
+    }
+
+    if (this.currencyFormatter) {
+      const containsNumber = /\d/g;
```

Comment:
does this regex check if a value HAS a number, or if it ONLY is a number (and no letters?). because then the name is a bit misleading.

Reply:
HAS. Because only if input contains a number the formatter can work correctly

Reply:
what about `const isNumber`?

Reply:
Do not change the variable, name is perfect.